pkg/api/validation/validation.go

```diff
@@ -52,6 +52,7 @@ const fieldImmutableErrorMsg string = `field is immutable`
 const isNotIntegerErrorMsg string = `must be an integer`
 
 var pdPartitionErrorMsg string = validation.InclusiveRangeError(1, 255)
+var volumeModeErrorMsg string = validation.InclusiveRangeError(0, 0777)
```

Comment:
I would not use this error.  This will print integer value, which is not friendly.  I would say something like "must be less than or equal to 0777" or "may only include bit in 0777" or simply "must be a valid permissions mode"